AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -225,6 +225,7 @@ - (void)completeSignInWithErrorCode:(NSInteger)errorCode andMessage:(NSString *)
                                               code:errorCode
                                           userInfo:@{NSLocalizedDescriptionKey : errorMessage}];
   self.signInCompletionHandler(nil, error);
+  self.signInCompletionHandler = nil;
```

Comment:
Don't you also need this line below line 188 [here](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1458/files#diff-eb504461a7342631bb7a7693117de49cR188)?

Reply:
Never mind. It doesn't assign the completionHandler when it is disabled so it is fine.